From 5fe4d9ff161a81898e0edfc7a77aa8ed71fe1e62 Mon Sep 17 00:00:00 2001
From: Ryan Volz <rvolz@mit.edu>
Date: Wed, 20 May 2020 11:41:55 -0400
Subject: [PATCH 1/3] Don't build a resource file for the executable on
 Windows.

conda-forge is not set up with an RC compiler.
---
 src/CMakeLists.txt | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index ffe3220..4e5a2c9 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -42,19 +42,6 @@ QT5_ADD_RESOURCES(RESOURCES_LIST
     ../resources/textfiles.qrc
 )
 
-########################################################################
-# Resource file - adds an icon to GQRX executable
-if (WIN32)
-    set(ICON_SOURCE ${CMAKE_SOURCE_DIR}/resources/icons/gqrx.ico)
-    set(RES_FILES "${CMAKE_CURRENT_BINARY_DIR}/gqrx.rc")
-    file(WRITE "${RES_FILES}" "id ICON \"${ICON_SOURCE}\"")
-    set(CMAKE_RC_COMPILER_INIT windres)
-    enable_language(RC)
-    set(CMAKE_RC_COMPILE_OBJECT
-        "<CMAKE_RC_COMPILER> <FLAGS> -O coff <DEFINES> -i <SOURCE> -o <OBJECT>")
-    list(APPEND RESOURCES_LIST ${RES_FILES})
-endif(WIN32)
-
 #######################################################################################################################
 # Build the program
 add_executable(${PROJECT_NAME} ${${PROJECT_NAME}_SOURCE} ${UIS_HDRS} ${RESOURCES_LIST})
-- 
2.20.1

